### ISSUE TITLE
0crat doesn't resign job performer after 10 days.

### ISSUE THREAD

**Issue by victornoel** [zerocracy/farm#2181]
I'm not clear if it's a feature or a bug, but it looks like a bug to me.

See https://github.com/yegor256/cactoos/issues/1184#issuecomment-541337919

Basically, the task has been assigned for a month and 0crat didn't resign the user from it after 10 days as it should.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@victornoel it was implemented intentionally #722:
```java
  if (new Ledger(farm, project).bootstrap().cash().decimal() <= BigDecimal.ZERO) {
    // We must not resign when the project is not funded, simply
    // because developers can't do anything without money. Their PRs
    // will have no reviewers, etc.
    return
  }
```
but feel free to submit a bug in policy repo to improve documentation about that case.

**Comment by 0crat** [comment]
This job is not in scope
